apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -287,7 +305,7 @@ function useNavItems(): Record< string, Post[] > {
 		const statuses = select( 'core' ).getEntityRecords( 'root', 'status' );
 
 		if ( ! statuses ) {
-			return [];
+			return { current: [], drafts: [], recent: [] };
```

Comment:
The original code was never checking for an empty array anyways -- it always assumed that these properties existed. This is a little safer then, plus fixes the type issues here.